test/transcoding/SPV_INTEL_media_block_io.cl

```diff
@@ -0,0 +1,214 @@
+// RUN: %clang_cc1 -triple spir-unknown-unknown -O1 -cl-std=CL2.0 -finclude-default-header -emit-llvm-bc %s -o %t.bc
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_media_block_io -spirv-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_media_block_io -o %t.spv
+// RUN: spirv-val %t.spv
+// RUN: llvm-spirv -r --spirv-ext=+SPV_INTEL_media_block_io %t.spv -o %t.rev.bc
```

Comment:
```suggestion
// RUN: llvm-spirv -r %t.spv -o %t.rev.bc
```

All extensions are considered as "allowed" during consumption step

Reply:
Done. Thanks for the suggesiton.